app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,15 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+    @post = Post.find(params[:post_id])
+  end
+
+  def create
+    @user = current_user
+    @post = Post.find(params[:post_id])
+    @comment = Comment.new(author_id: @user.id, post: @post, text: params[:comment][:text])
+    @comment.save
+
+    redirect_to user_posts_path(@user, @post)
```

Comment:
- [Optional] Same here, As mentioned earlier, consider using optional redirect based on the success and failure of the create action.

Reply:
Thank you @TadesseDev  I will do accordingly.